presto-main/src/main/java/com/facebook/presto/operator/JoinProbeBase.java

```diff
@@ -13,7 +13,25 @@
  */
 package com.facebook.presto.operator;
 
+import com.facebook.presto.spi.Page;
+import com.facebook.presto.spi.PageBuilder;
+import com.facebook.presto.spi.block.Block;
+import com.facebook.presto.spi.block.DictionaryBlock;
+
 public abstract class JoinProbeBase
     implements JoinProbe
 {
+    protected Page internalBuildDictionaryPage(int[] indices, int[] probeOutputChannels, PageBuilder sourcePageBuilder)
```

Comment:
Are this `indices` within probe page or within "sourcePageBuilder.build()" page?